### ISSUE TITLE
AttributeAtom::toRelationAtom will fail if the attribute is only used as a key

### ISSUE THREAD

**Issue by kasper-piskorski** [graknlabs/grakn#5503]
## Description

Atom conversion from `AttributeAtom` to `RelationAtom` doesn't take into account the existence of keys.
## Reproducible Steps
Prepare:
```
define
someEntity sub entity, key resource;
resource sub attribute, datatype string;

insert
$x isa someEntity, has resource 'value';
```

Run:
```
ReasonerQueries.atomic(conjunction("{ $x has resource; };", tx), tx).getAtom().toRelationAtom();
```

## Expected Output

Relation equivalent of the attribute atom in question corresponding to the attribute implicit relation.

## Actual Output
`NullPointer` exception.

**Comment by flyingsilverfin** [comment]
This will no longer be needed after removing the implicit attribute relations, so we just ignore it for now. #5614

**Comment by flyingsilverfin** [comment]
Closed, as of 1.8 we no longer have implicit relations
